### ISSUE TITLE
Consider creating graphscope-client-nightly on PyPI

### ISSUE THREAD

**Issue by TechCiel** [alibaba/GraphScope#2366]
Hi,

I'm currently maintaining a public mirror of PyPI, and noticed the high and frequent traffic syncing graphscope-client.

As a prior art, the tensorflow project use packages like tf-nightly to publish nightly versions. As every nightly version costs hundreds of megabytes to sync to mirrors over the world, would you please consider use graphscope-client-nightly for nightly builds as a way for mirrors to filter based on names, and for users to select the nightly version more conveniently?

Thanks in advance for any cooperation.

TL;DR in zh_CN: 可以学学 tensorflow ，用 graphscope-client-nightly 发布每日构建的 python 包吗？一个好几百兆，公开镜像要按名过滤的。

**Comment by sighingnow** [comment]
We do have nightly release on pypi: https://pypi.org/project/graphscope/#history

**Comment by sighingnow** [comment]
> We do have nightly release on pypi: https://pypi.org/project/graphscope/#history

The nightly build can be installed by `pip3 install -U graphscope --pre`.

**Comment by TechCiel** [comment]
@sighingnow, I'd humbly ask you to see this again, as I'm not asking for nightly builds, but for the nightly builds to be published separately on `graphscope-*-nightly`s instead of the main packages. This will provide PyPI mirrors a possibility to filter out nightly builds.

**Comment by sighingnow** [comment]
No such plan.

Is there anything that `--pre` doesn't work and a seperate package name would work?

**Comment by sighingnow** [comment]
After internal discussion we still have no plan for a separate package name and we believe delivering nightly build to end users is important for us for early feedback.

To optimize the traffic need to sync the package, we're consider:

1. optimize package size, I think there still be a large room for that
2. change the pre-release build cycle as weekly rather than nightly.
